packages/types/src/context-types.ts

```diff
@@ -5,7 +5,7 @@ import { Spread, TuplifyUnion, Unarray } from './utils';
 // We are using `interface` instead of `type` in order to allow type augmentation
 export interface DefaultContext extends Record<string | symbol | number, unknown> {}
 
-export type ComposeContextArray<V extends unknown> = V extends []
+export type ComposeContextArray<V> = V extends []
```

Comment:
Constraining the generic type `V` to `unknown` does nothing and is unnecessary  @typescript-eslint/no-unnecessary-type-constraint